metrics.go

```diff
@@ -2,7 +2,12 @@ package main
 
 import 	(
 	"github.com/rcrowley/go-metrics"
+	"github.com/cyberdelia/go-metrics-graphite"
+	"net"
 	"net/http"
+	"time"
+	standardLog "log"
+	"os"
```

Comment:
It looks like that you did not format the source code properly. 
I suggest you to use